test/models/tag_test.js

```diff
@@ -0,0 +1,242 @@
+import { mkdbtmp } from '../support/db'
+
+import { project as projectModel, item as itemModel, tag as tagModel } from '../../src/models'
+
+
+describe('models', async () => {
+  describe('tag', () => {
+    let db
+
+    mkdbtmp(x => db = x,
+      'db_test.sqlite', projectModel.create, { name: 'Test Project' })
```

Comment:
Here is a good illustration about the potential dangers of using `this` in the model functions. If we use `this` in `create` here it will fail.